apps/wing/src/cli.ts

```diff
@@ -211,6 +211,14 @@ async function main() {
     .hook("preAction", collectAnalyticsHook)
     .action(runSubCommand("pack"));
 
+  program
+    .command("new")
+    .description("Create a new Wing project")
+    .argument("[template]", "Template name")
+    .addOption(new Option("-l --language [language]", "Language"))
+    .hook("preAction", collectAnalyticsHook)
+    .action(runSubCommand("init"));
+
```

Comment:
I wonder if it would make sense to use commander `choices` to maintain the available templates.

Reply:
I went ahead and added the `choices` so the list of templates appear on the `--help` page. I skipped this initially because commander would just throw an opaque error to the user like `missing required argument 'template'` which didn't seem ideal. But it looks like I found a way so I can still give custom errors / provide an interactive menu, so that seems like the best of both worlds.